### ISSUE TITLE
Support for Apache Arrow/Feather? 

### ISSUE THREAD

**Issue by rdong8** [bluenote10/NimData#63]
Is there support for the Apache Feather V2 format? I believe Python's Pandas supports them.

**Comment by mratsim** [comment]
No, there is no support at the moment.

**Comment by bluenote10** [comment]
Yes, back then when I considered adding it, Arrow was lacking C bindings which made it tricky. It looks like [Arrow now has C bindings](https://arrow.apache.org/blog/2020/05/03/introducing-arrow-c-data-interface/) which would make this more feasible I guess.
